covalent_braket_plugin/braket.py

```diff
@@ -372,14 +471,17 @@ def _poll_braket_job(self, braket, job_arn: str) -> None:
             None
         """
 
-        status = self.get_status(braket, job_arn)
+        status = await self.get_status(braket, job_arn)
 
+        loop = asyncio.get_running_loop()
         while status not in ["COMPLETED", "FAILED", "CANCELLED"]:
             time.sleep(self.poll_freq)
```

Comment:
@cjao last small change here, let's try to use `asyncio.sleep` rather than the blocking `time.sleep`

Reply:
Oops, thanks for catching that!